### DISCUSSION TITLE
Using `end if` as an alias for `endif` in Activity diagrams

### DISCUSSION THREAD

**Discussion by rohitsud** [plantuml/plantuml#1311] [Ideas]
At many places in PlantUml, we use `end <object>` syntax to close a node. For example, `end note`. However, for `if` statements in activity diagrams, only `endif` is supported. The proposal is to also support `end if` as an alias for `endif` and potentially deprecate `endif` at a later stage.

**Comment by arnaudroques** [comment]
Very good idea!
This has been implemented in last snapshot and on the online server.
Thanks for the suggestion
